src/viam/sdk/services/private/mlmodel_client.cpp

```diff
@@ -36,13 +46,15 @@ std::shared_ptr<MLModelService::named_tensor_views> MLModelServiceClient::infer(
     namespace mlpb = ::viam::service::mlmodel::v1;
 
     auto arena = std::make_unique<pb::Arena>();
-    auto* const req = pb::Arena::CreateMessage<mlpb::InferRequest>(arena.get());
+    auto* const req = pb::Arena::VIAM_SDK_PB_CREATE_MESSAGE<mlpb::InferRequest>(arena.get());
 
     req->set_name(this->name());
     *req->mutable_extra() = map_to_struct(extra);
-    auto* const resp = pb::Arena::CreateMessage<mlpb::InferResponse>(arena.get());
+    auto* const resp = pb::Arena::VIAM_SDK_PB_CREATE_MESSAGE<mlpb::InferResponse>(arena.get());
     ClientContext ctx;
 
+#undef VIAM_SDK_PB_CREATE_MESSAGE
```

Comment:
Personally, I'd be OK with not bothering to undefine here in .cpp context since the macro isn't going to leak and conflicts arising later in the file seem very unlikely.

Reply:
I think you're right, just removed it